integration_test/chrome/capabilities_test.exs

```diff
@@ -1,13 +1,14 @@
 defmodule Wallaby.Integration.CapabilitiesTest do
-  use ExUnit.Case
+  use ExUnit.Case, async: false
   use Wallaby.DSL
+
+  import Wallaby.SettingsTestHelpers
+
   alias Wallaby.Integration.SessionCase
   alias Wallaby.Experimental.Selenium.WebdriverClient
 
   setup do
-    on_exit(fn ->
-      Application.delete_env(:wallaby, :chromedriver)
-    end)
+    ensure_setting_is_reset(:wallaby, :chromedriver)
```

Comment:
@mhanberg This change isn't actually required for this PR (since #512 fixed the chromedriver failures we seeing), but it seems to be more correct to just reset the environment instead of completely deleting any existing settings. Do you mind if I leave this in here?

Reply:
Fine by me